src/KafkaW/Consumer.h

```diff
@@ -9,21 +9,32 @@
 
 namespace KafkaW {
 
-class Inspect;
+class ConsumerInterface {
+public:
+  ConsumerInterface() = default;
+  virtual ~ConsumerInterface() = default;
+  virtual void addTopic(std::string Topic,
```

Comment:
virtual methods can't have default arguments

Reply:
Turns out that they can but they do not do anything in this case. I have removed the default argument.